### ISSUE TITLE
Random-ish build failure around preprocessor_successor.h

### ISSUE THREAD

**Issue by chrysn** [RIOT-OS/rust-riot-sys#35]
There are build errors cropping up with:

```
   --- stderr
  /__w/rust-riot-wrappers/rust-riot-wrappers/RIOT/sys/include/auto_init_utils.h:35:10: fatal error: 'preprocessor_successor.h' file not found
  thread 'main' panicked at /github/home/.cargo/git/checkouts/rust-riot-sys-d12733b89271907c/c985a6b/build.rs:224:10:
  Unable to generate bindings: ClangDiagnostic("/__w/rust-riot-wrappers/rust-riot-wrappers/RIOT/sys/include/auto_init_utils.h:35:10: fatal error: 'preprocessor_successor.h' file not found\n")
```

I've now traced this down to being caused by compile-commands.json files where the `-I.../bin/native/preprocessor` entry is not in the first of the included files.

**Comment by chrysn** [comment]
More precisely, this is due to the consensus cflags building process: Generally, -I flags are always kept (`keep |= i[0].starts_with("-I")`), but that makes the first entry special.

**Comment by chrysn** [comment]
Closed through #36.
